modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js

```diff
@@ -83,6 +91,8 @@ export default ({section}) => {
 
 	return (
 		<section className="align-items-center d-flex">
+			{visible &&
+				<NewTopicModal/>}
```

Comment:
I think functions are not exactly the same as components and are re-initialized every time the render function is called, read this: https://stackoverflow.com/questions/46777832/react-noob-onchange-element-loses-focus/46791156#46791156

Reply:
I am not sure if I understand what do you mean. The linked comment says that `Because FieldGroup(their function) is declared inside render, a new function is created on each rendering` but in our case, the function _NewTopicModal_ is not declared inside the render so, the behavior is the same? The function is re-initialized?

My point about the problem with the infinite rendering is that if the visible condition is above the _useModal_ hook, it may be lose
This is documented [here](https://clayui.com/docs/components/modal.html#usage) just after the example.

This problem should be solved in the next commits following the previous documentation.

Reply:
I think the behaviour is the same, because it's created inside the component. Functions inside the component are created every time, that's why React recommends using useCallback to avoid it. Can you try it calling it as a function like `{LocalSFC({rerender: this.rerender})}` instead of <LocalSFC ???

Or inline <NewTopicModal or extract it to a real component.

I'm going to merge it, but try this ^